test/semantics/CMakeLists.txt

```diff
@@ -91,6 +91,7 @@ set(ERROR_TESTS
   computed-goto02.f90
   nullify01.f90
   nullify02.f90
+  stop.f90
```

Comment:
Please adopt the 0n naming (stop01, ...) in case we add another test for stop.

Reply:
Done.